apps/ai-content-generator/package.json

```diff
@@ -54,13 +54,13 @@
     "@typescript-eslint/eslint-plugin": "^6.2.1",
     "@typescript-eslint/parser": "^6.2.1",
     "@vitejs/plugin-react": "4.0.4",
+    "@vitest/coverage-v8": "0.34.6",
```

Comment:
I like the addition of seeing coverage while running the tests!

Reply:
Thanks! It's such a great tool. Seeing the % gives me joy